.github/workflows/docker-compose-build.yml

```diff
@@ -87,7 +87,7 @@ jobs:
           CMD="go version | awk '{print \$3}' | sed -e 's/^go//g'"
           go_version=$(docker compose -f docker-compose.yml -f dev.docker-compose.yml run deploy-webhook sh -c "${CMD}")
           echo "Go version:" "${go_version}"
-          echo "::set-output name=go_version::${go_version}"
+          echo "go_version=${go_version}" >> $GITHUB_OUTPUT
```

Comment:
https://github.com/dev-hato/deploy-webhook/actions/runs/3243580534/jobs/5318501548

```
.github/workflows/docker-compose-build.yml:86:9: shellcheck reported issue in this script: SC2086:info:4:36: Double quote to prevent globbing and word splitting [shellcheck]
   |
86 |         run: |
   |         ^~~~
```

Reply:
```suggestion
          echo "go_version=${go_version}" >> "${GITHUB_OUTPUT}"
```